helpers/src/db/pagination.rs

```diff
@@ -0,0 +1,69 @@
+use diesel::pg::Pg;
+use diesel::prelude::*;
+use diesel::query_builder::*;
+use diesel::query_dsl::methods::LoadQuery;
+use diesel::sql_types::BigInt;
+use diesel::QueryId;
+
+const DEFAULT_PER_PAGE: i64 = 10;
+
+pub trait Paginate: Sized {
```

Comment:
Is `Sized` really the only prerequisite? I would assume we should restrict it to `LoadQuery<PgConnection, (U, i64)>` (from line 38), because else the most important method won't work.

Reply:
Yes, I think so. The `Paginated` struct contains the query as boxed.